src/transaction/log_manager.c

```diff
@@ -12449,6 +12449,26 @@ cdc_make_error_loginfo (int trid, char *user, CDC_DML_TYPE dml_type, OID classoi
     } \
   while (0)
 
+static bool
```

Comment:
Please leave a comment about the followings. So let other developers know why you don't service if there is a change of any schema.
1. cdc - design issue (Issues due to the difference between the time the client obtains schema information through jdbc and the time it takes to interpret log records)
2. flashback - cannot flashback even if you generate any SQL formatted by old schema

Reply:
OK, I will add the comment for it.